device/components/RTA_OnDeviceComponentTask.brs

```diff
@@ -270,19 +417,58 @@ sub sendBackError(request as Object, message as String)
 	sendBackResponse(request, buildErrorResponseObject(message))
 end sub
 
-sub sendBackResponse(request as Object, response as Dynamic)
+sub sendBackResponse(request as Object, response as Object, binaryPayloadByteArray = Invalid as Dynamic)
```

Comment:
the word `back` is unnecessary. what about calling it `sendResponse` instead?

Reply:
For this one I think maybe we could name it `sendResponseToClient`?

Reply:
Yeah, I like that better.